index.yaml

```diff
@@ -1855,6 +1855,36 @@ entries:
     - assets/fleet-crd/fleet-crd-0.3.000.tgz
     version: 0.3.000
   harvester-cloud-provider:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Harvester Cloud Provider
+      catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.30.0-0'
+      catalog.cattle.io/namespace: kube-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/permits-os: linux
+      catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+      catalog.cattle.io/release-name: harvester-cloud-provider
+      catalog.cattle.io/ui-component: harvester-cloud-provider
+      catalog.cattle.io/upstream-version: 0.2.4
+    apiVersion: v2
+    appVersion: v0.2.1
+    created: "2024-05-07T18:23:43.582162+08:00"
+    dependencies:
+    - name: kube-vip
+      repository: file://./charts/kube-vip
+      version: 0.4.2
```

Comment:
@FrankYang0529 Comparing to another 2 PRs, `0.4.2` is still existing in index.yaml;

it is due to `Chart.yaml.patch` has no `-` line to remove this version, please double check, thanks.